src/Simplex/Chat/Store/Messages.hs

```diff
@@ -315,7 +315,7 @@ updateChatTs db User {userId} chatDirection chatTs = case toChatInfo chatDirecti
   DirectChat Contact {contactId} ->
     DB.execute
       db
-      "UPDATE contacts SET chat_ts = ? WHERE user_id = ? AND contact_id = ?"
+      "UPDATE contacts SET chat_ts = ?, chat_deleted = 0 WHERE user_id = ? AND contact_id = ?"
```

Comment:
on new chat items UI would "manually" update contacts with chatDeleted set to False. May be a bit hacky but it seems a normal assumption to do from UI view point and less hassle than correctly propagate contact update from db operations to all responses in backend. This is somewhat similar in logic to chat "popping" to top on new chat items being only UI logic